### DISCUSSION TITLE
Autoplay videos in preview

### DISCUSSION THREAD

**Discussion by vladstudio** [mikekazakov/nimble-commander#253] [Ideas]
When previewing multiple videos in a folder, I have to click Play button every time. It'd be more convenient to have videos autoplay.

Demo:

https://github.com/mikekazakov/nimble-commander/assets/914320/626a3903-3940-4af2-8d14-34e0fc319099

Maybe makes sense to make it a checkbox in settings.

**Comment by mikekazakov** [comment]
I'm not entirely sure if that's possible, but I'd really appreciate any guidance or tips...

**Comment by gingerbeardman** [comment]
Would like the same for audio previews.

Workaround, I am using the option to just open QuickLook panel.
